ReShade/Presets/Default/Shaders_by_Iddqd.cfg

```diff
@@ -42,3 +42,15 @@
 #define DaltonizeFX_Type 1 //[1:3] //-1 = Protanopia, 2 = Deuteranopia, 3 = Tritanopia
 #define DaltonizeFX_ToggleKey RESHADE_TOGGLE_KEY //[undef] //-
 
+////---------------//
+///**GRADIENTFX**///
+//---------------////
+#define USE_GRADIENTFX  0 //[GradientFX] //-Gradient fx
+#define GradientTop float3(0.5,0.0,1.0) //[0.00:1.00] //-No description given
+#define GradientBottom float3(1.0,1.0,0.0) //[0.00:1.00] //-No description given
+#define GradientShift 0.5 //[0.0:1.0]
+#define GradientBlendPower 0.0 //[0.0:2.0]
+#define GradientOpacility 0.1  //[0.0:1.0]
+#define GradientRotation radians(90.0) - radians(180.0) //[undef] //-Settings for rotation
```

Comment:
As far as I know the assistant tool is unable to correctly handle spaces currently, so this only works without spaces around the minus operator.